gitlab/v4/objects.py

```diff
@@ -4712,6 +4712,74 @@ def import_project(
             "/projects/import", post_data=data, files=files, **kwargs
         )
 
+    def import_github(
+        self,
+        personal_access_token,
+        repo_id,
+        target_namespace,
+        new_name=None,
+        timeout_override=60.0,
+        **kwargs
+    ):
+        """Import a project from Github to Gitlab (schedule the import)
+
+        This method will return when an import operation has been safely queued,
+        or an error has occurred. After triggering an import, check the
+        `import_status` of the newly created project to detect when the import
+        operation has completed.
+
+        NOTE: this request may take longer than most other API requests.
+        So this method will override the session timeout with <timeout_override>,
+        which defaults to 60 seconds.
+
+        If repo_id is specified as an integer, it will be used directly.
+        If repo_id is a string, we will attempt to look it up using PyGithub.
+        The preferred string format is <org>/<repository>. However the python
+        package `PyGithub` must be installed to perform this lookup.
```

Comment:
The docs will need some updates

Reply:
Whoops! Resolved in 89d8f2b253803923c9fd633a11df66d409709cda.